test/e2e-test/pages/helper-functions.ts

```diff
@@ -14,7 +14,7 @@ async function signInHelper() {
 
 function signInForUser(email: string) {
   I.fillField('#username', email);
-  I.click('.button');
+  I.click('Sign in');
```

Comment:
Didn't we try this and it broke something. Much better to use Sign in but have a feeling this doesn't work when you run the e2e tests. But guess we commit and see would be good to work out why if this doesn't work.

Reply:
I think it was failing because of the login button text in the mockup page was something like 'submit' since the button now says 'Sign in' this should be working. Also, functional tests are passing.